functions/weather.fish

```diff
@@ -49,15 +49,17 @@ function weather -d "Displays weather info"
   set wind_deg (echo $json | jq '.wind.deg')
 
   # convert m/s to km/h
-  set wind_speed_kmh ( math -s 1 $wind_speed \* 3.6 )
-  set  wind_gust_kmh ( math -s 1  $wind_gust \* 3.6 )
+  set wind_speed_kmh ( math -s 1 "$wind_speed * 3.6" )
+  if not test $wind_gust = null
+    set wind_gust_kmh ( math -s 1 "$wind_gust * 3.6" )
+  end
 
   # If we want miles per hour
-  #set wind_speed_mph ( math -s 1 $wind_speed \* 2.23694 )
+  #set wind_speed_mph ( math -s 1 "$wind_speed * 2.23694" )
 
   # Get the cardinal direction from the heading
   set directions N NE E SE S SW W NW N
-  set wind_dir $directions[(math "(($wind_deg % 360) / 45) + 1")]
+  set wind_dir $directions[(math -s0 "(($wind_deg % 360) / 45) + 1")]
```

Comment:
Don't truncate. Round instead:

```suggestion
  set wind_dir $directions[(math "round(($wind_deg % 360) / 45) + 1")]
```

This:

```
echo $directions[(math "round((89 % 360) / 45) + 1")]
E
```

Is a bit more accurate than this:

```
echo $directions[(math -s0 "((89 % 360) / 45) + 1")]
NE
```